content/tutorial-escrow.md

```diff
@@ -19,6 +19,406 @@ When testing in [stand-alone mode](concept-stand-alone-mode.html), you can force
 You can check the status of the Escrow amendment using the [`feature` command](reference-rippled.html#feature).
 
 
-## TODO
+## Send a Time-Held Escrow
+
+The [EscrowCreate transaction][] type can create an escrow whose only condition for release is that a specific time has passed. To do this, use the `FinishAfter` field and omit the `Condition` field.
+
+### 1. Calculate release time
+
+You must [specify the time](reference-rippled.html#specifying-time) as whole **seconds since the Ripple Epoch**, which is 946684800 seconds after the UNIX epoch. For example, to release funds at midnight UTC on November 13, 2017:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*JavaScript*
+
+```js
+// JavaScript Date() is natively expressed in milliseconds; convert to seconds
+const release_date_unix = Math.floor( new Date("2017-11-13T00:00:00Z") / 1000 );
+const release_date_ripple = release_date_unix - 946684800;
+console.log(release_date_ripple);
+// 563846400
+```
+
+<!--{# //Python code works OK but we don't have full examples, so hiding it
+*Python 3*
+
+```python
+import datetime
+release_date_utc = datetime.datetime(2017,11,13,0,0,0,tzinfo=datetime.timezone.utc)
+release_date_ripple = int(release_date_utc.timestamp()) - 946684800
+print(release_date_ripple)
+# 563846400
+```
+
+#}-->
+
+<!-- MULTICODE_BLOCK_END -->
+
+**Warning:** If you use a UNIX time in the `FinishAfter` field without converting to the equivalent Ripple time first, that sets the unlock time to an extra **30 years** in the future!
+
+### 2. Submit EscrowCreate transaction
+
+[Sign and submit](reference-transaction-format.html#signing-and-submitting-transactions) an [EscrowCreate transaction][]. Set the `FinishAfter` field of the transaction to the time when the held payment should be released. Omit the `Condition` field to make time the only condition for releasing the held payment. Set the `Destination` to the recipient, which may be the same address as the sender.
+
+{% include 'snippets/secret-key-warning.md' %}
+
+Request:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*Websocket*
+
+```json
+{% include 'code_samples/escrow/websocket/submit-request-escrowcreate-time.json' %}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+Response:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*Websocket*
+
+```json
+{% include 'code_samples/escrow/websocket/submit-response-escrowcreate-time.json' %}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+
+Take note of the transaction's identifying `hash` value so you can easily check its final status when it is included in a validated ledger version.
+
+### 3. Wait for validation
+
+{% include 'snippets/wait-for-validation.md' %}
+
+### 4. Confirm that the escrow was created
+
+Use the [`tx` command](reference-rippled.html#tx) with the transaction's identifying hash to check its final status. Look for a `CreatedNode` in the transaction metadata to indicate that it created an [Escrow ledger object](reference-ledger-format.html#escrow).
```

Comment:
Do you want to indicate how to look in the transaction metadata?

Reply:
It's a bit complicated; we have a task (DOC-123) to write a full explanation of how to read tx metadata.